codes/classical/q-ary_digits/justesen.yml

```diff
@@ -0,0 +1,118 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# Use UTF-8 unicode encoding
+# AMS-TeX commands are rendered inside \( ... \) using MathJaX.
+# Allowed external bibliographic references are
+#    \cite{arXiv:#.#} or \cite{arXiv:quant-ph/#} (PREFERRED),
+#    \cite{doi:#}, or, as a last resort
+#    \cite{manual:{(enter citation line incl. author and year here)}}
+# External websites such as code tables, coding theory packages, github pages linked as
+#    \url{https://example.com/example}
+#    \href{https://example.com/example}{link text}
+# Internal references to codes are
+#    \hyperref[code:code_id]{link text}
+# Delete instructional comments when submitting
+
+# code id, physical, logical are all lower case
+code_id: justesen
+physical: q-ary_digits
+logical: q-ary_digits
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title (SINGULAR) + first reference(s) (optional).
+name: 'Justesen_code'
+introduced: '\cite{manual:{MacWilliams/Sloane, The Theory of Error-Correcting Codes, Sec.10.11}}'
+
+# Anything applicable to a larger parent set of codes (see below) should go in
+# that entry instead of here.
+description: 'Given m, Justesen_code is a binary code over GF(2^m), with length n=2mN, dimension k=mK, where (N=2^m-1,K) is the reed-solomon outer code.
+              The code can be considered as concatenation of a Reed-Solomon outer code with N distinct inner codes.'
```

Comment:
The code can be considered as concatenation of a Reed-Solomon outer code **of length N over F_{2^m}** with N distinct inner codes.'